src/odemis/gui/cont/streams.py

```diff
@@ -1638,7 +1731,8 @@ def _add_stream_cont(self, stream, show_panel=True, locked=False, static=False,
         """
 
         stream_cont = StreamController(self._stream_bar, stream, self._tab_data_model,
-                                       show_panel, view, self._view_controller)
+                                       show_panel, view, self._view_controller,
+                                       show_md=self.static_mode)  # show metadata if stream is static
```

Comment:
The problem with just deciding on "static_mode" is that it might trigger in some unwanted cases. For example, the SECOM/DELPHI acquisition window has this mode set. I'd suggest to get read of the `show_md` argument, and do it automatically when the stream is a StaticStream.  I'm not 100% sure it will work properly in every case, so you'll still want to double check.